tests/CMakeLists.txt

```diff
@@ -1,15 +1,22 @@
 # Require at least version 2.8.12 of CMake
 cmake_minimum_required(VERSION 2.8.12)
 
-# Include global configure settings
-include(${CMAKE_CURRENT_SOURCE_DIR}/../configure.cmake)
-include(${CMAKE_CURRENT_SOURCE_DIR}/../configurecompiler.cmake)
+if (NOT WIN32)
+  set(CMAKE_CXX_FLAGS_INIT "${CMAKE_CXX_FLAGS_INIT} -std=c++11"  CACHE STRING "Flags used by the compiler during all build types.")
+  set(CMAKE_C_FLAGS_INIT "${CMAKE_C_FLAGS_INIT} -std=c11"  CACHE STRING "Flags used by the compiler during all build types.")
+endif()
+
+project(Project)
```

Comment:
Was adding this necessary?

Reply:
For the `*_INIT` properties to work correctly, the languages need to be enabled after setting them. I can try some variations here (using just the `CMAKE_<LANG>_FLAGS` properties instead of the init variants or using `enable_language()`) if we don't want to have the `project()` call.